### ISSUE TITLE
Hot reload skin.ini while watching autoplay

### ISSUE THREAD

**Issue by Jolomann** [Quaver/Quaver#4040]
### Is your feature request related to a problem? Please describe.

TL;DR: Make skin.ini actively reload like tournament settings.ini but limit to autoplay replays

Going back to song select, reloading skin, and then going back into autoplay to adjust things in a skin can get tedious. Tournament settings.ini solves this by actively reloading on file change. Having the same functionality for skin.ini would be crazy QoL

### Describe the solution you'd like

For skin.ini, hot reload won't be necessary all the time, and it may cause performance issues while playing, so limiting it to replays would make more sense. Specifically autoplay replays might be the way to go, so skinners can benefit from this without the feature potentially impacting people on low end hardware capturing their actual replays.
